taxi/views.py

```diff
@@ -87,3 +89,37 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
```

Comment:
Consider using `get_user_model()` instead of directly referencing the `Driver` model. This will make your code more flexible and aligned with Django best practices.